### PR TITLE
Discovered art page

### PR DESCRIPTION
**Pull request by gwholland3**
## Summary
Added UI elements to mirror the Framer mockup, visit the page `artwork/1` to view. Also changed some overarching styling.

## Todo
Still missing the scrolling pictures, and button functionality.

### CODE REVIEW

**Comment by gwholland3** [comment]
Fixes #25.

File: frontend/src/components/Tag/Tag.scss [hunk 1]
```diff
@@ -1,12 +1,17 @@
 .Tag {
   display: inline-block;
-  margin: 5px;
-  font-size: 13px;
-  font-weight: 600;
-  padding: 10px 24px;
+  margin: 2px;
+  font-size: 15px;
+  font-weight: 500;
+  padding: 4px 17px;
```
Comment (Waidhoferj) [inline frontend/src/components/Tag/Tag.scss]: I'm not so sure about these changes. At minimum we should have a larger margin between the Tag elements. Check out the map page which uses the Tag component. The elements are too close together.

File: frontend/src/index.css [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 :root {
   --c-background: white;
-  --c-primary: #3288ff;
+  --c-primary: #32A5ff;
```
Comment (Waidhoferj) [inline frontend/src/index.css]: Why are we changing the primary color here?

**Review by Waidhoferj — CHANGES_REQUESTED** [review]
There are a few style changes that are cascading across the project. Did you mean to make these changes?

**Comment by gwholland3** [comment]
> There are a few style changes that are cascading across the project. Did you mean to make these changes?

Yeah I was just trying to get it to look almost identical to the tags in the Framer mockup. Should I keep them the way they're styled globally? Or is there an easy way to apply those changes just to the tags (and buttons in the case of the color) on the discovered art page?

**Comment by Waidhoferj** [comment]
Screen Shot 2021-03-03 at 6 36 36 PM
Screen Shot 2021-03-03 at 6 38 49 PM
@gwholland3 I think the original dimensions were closer to the ground truth

**Comment by gwholland3** [comment]
Ok I'll restore them 👍

**Comment by gwholland3** [comment]
Ok so I was resetting most of the css properties, but I feel like at the very least a font size of 13 pixels is too small, and the font weight should probably be 500, since at 600 the letters are basically the same thickness as the border. Font size of 16 or 17 makes the relative sizing look more like the mockup imo.

**Comment by Waidhoferj** [comment]
That works great! Good call!

**Review by kyleshare — APPROVED** [review]
Looks good

**Review by Waidhoferj — APPROVED** [review]
Looks good

**Comment by Waidhoferj** [comment]
The branch is gonna need a rebase, it's all set once that's been done

**Comment by gwholland3** [comment]
@Waidhoferj What do you mean? Isn't it going to get merged into main?

**Comment by Waidhoferj** [comment]
Yeah I mean you have to resolve your merge conflicts. Usually you rebase and ensure everything works still and then do a clean merge.

**Review by Braden50 — APPROVED** [review]
All looks great!

**Pull request merged**